runtime/gc_base/RootScanner.cpp

```diff
@@ -227,6 +230,17 @@ MM_RootScanner::doStringTableSlot(J9Object **slotPtr, GC_StringTableIterator *st
 	doSlot(slotPtr);
 }
 
+#if defined(J9VM_GC_ENABLE_DOUBLE_MAP)
+/**
+ * @todo Provide function documentation
+ */
+void
+MM_RootScanner::doDoubleMappedObjectSlot(J9Object *objectPtr, struct J9PortVmemIdentifier *identifier)
+{
+        doSlot((J9Object **)&objectPtr);
```

Comment:
I suspect this is unnecessary - we should just do nothing. 
Firstly, doSlot is passing in/out  parameter, and whatever output is created (say if object is moved and address has changed) it will be anyway lost since objectPtr is only in parameter in doDoubleMappedObjectSlot.
Secondly, we indeed don't need to update pointer to spine object (in case of RootScanner for Compactor), since that it's already done (see fixupArrayletLeafRegionSpinePointers).